### DISCUSSION TITLE
Correct version for initialVisibilityDelay > 7 Days

### DISCUSSION THREAD

**Discussion by Maverick1983** [Azure/azure-functions-durable-extension#2106] [Q&A] [answered]
Which version introduced timeout > 7 Days?

**Comment by cgillum** [comment, marked as answer]
Are you asking about removing the 7-day cap for Durable Timers? Looking through the release notes, it appears we added this in [v2.3.0](https://github.com/Azure/azure-functions-durable-extension/releases/tag/v2.3.0).

**Comment by Maverick1983** [comment]
Yes. Thanks.

Il Gio 10 Mar 2022, 19:20 Chris Gillum ***@***.***> ha
scritto:
